lib/shim/webframework-shim.js

```diff
@@ -677,6 +677,23 @@ function _defaultResponsePredicate() {
   return false
 }
 
+/**
+ * Attaches a flag on function indicating that
+ * CLM attributes need to be associated with the middleware
+ * span during record
+ *
+ * Note: director middleware instrumentation passes in a string as the "function" so we have to check if the middleware is actually a function
+ * to avoid crashing.
+ *
+ * @param {Shim} shim instance of shim
+ * @param {Function} middleware function to apply clm symbol
+ */
+function assignCLMSymbol(shim, middleware) {
+  if (shim.isFunction(middleware) && shim?.agent?.config?.code_level_metrics?.enabled) {
```

Comment:
woof 😬 
I feel like while it's nice that we have `?` that there should be some way to fetch values from our config without having to do stuff like this all over the place. Not asking you to change that here and now, just thinking out loud.

Reply:
i can change this now. this was done before I rebased with the config